common/hooks/useOnScreen.ts

```diff
@@ -35,11 +34,11 @@ export default function useOnScreen({
       },
       {
         root,
-        rootMargin,
+        rootMargin: '0px',
```

Comment:
I'm pretty sure this would be the default, so if we're not letting it be changed via a prop, I think we can remove the line altogether